src/MuTalk-Model/MutationTestingAnalysisBuilder.class.st

```diff
@@ -0,0 +1,136 @@
+Class {
+	#name : 'MutationTestingAnalysisBuilder',
```

Comment:
I wonder if here we really need a separate class.
We can have a builder pattern, but all in the same class.

The question is: what is the benefit of having a builder separate here?